app/misc/authenticate.ts

```diff
@@ -20,18 +20,69 @@ var repoNotFound = 0;
 var signed = 0;
 var changes = 0;
 let signedAfter = false;
-let loginScopes = [
-  "repo",
-  "user"
-];
 
+let oauthpass = 'x-oauth-basic';
+let electronOauth2 = require('electron-oauth2');
+let account;
+
+//config from oauth library
+const OauthConfig = {
+  clientId: 'c2509d8769f5f1e46028',
+  clientSecret: 'e4717832659e95c2f8f4237a3390cf09013b94f7',
```

Comment:
Put a /todo here to make sure to store the client secret in an environment variable if they ever rollout to production